modules/ROOT/assets/overlays/admin.yaml

```diff
@@ -0,0 +1,25 @@
+overlay: 1.0.0
+
+info:
+  title: Overlay for Admin REST API spec
+  version: 1.0.0
+
+extends: https://raw.githubusercontent.com/couchbase/sync_gateway/refs/heads/main/docs/api/admin.yaml
+
+actions:
+  - description: Update title and introduction
+    target: $.info
+    update:
+      title: Sync Gateway Admin REST API
+      description: |-
```

Comment:
I had been confused by this before, but I think that we want to update all the code we can in the sync gateway repo with the info + tags/descriptions, and then keep this overlay document with only:

```
  - description: Update link to external docs
    target: '$'
    update:
      externalDocs:
        description: ⬆️ Admin REST API Overview
        url: rest-api-admin.html
```

I am happy for you to do the PR against Sync Gateway for this, it would be against `main`. Then docs or SGW team can create backport to 3.2 branch, and we would move our tag.

Reply:
I think the intention might be to fix as part of https://jira.issues.couchbase.com/browse/CBG-4312 - I'd definitely consider putting that fix in before you do capella side of this.